app/views/api/clidocs/pept2prot.html.erb

```diff
@@ -214,6 +214,33 @@ ISVAQGASK,A0A087WXI2,IgGFc-binding protein,Homo sapiens,</pre>
     </p>
       </div>
     </div>
+    <div class='card'>
+      <div class='card-title card-title-colored'>
+        <h2 class='card-title-text' id='meganize'>Meganize</h2>
+      </div>
+      <div class='card-supporting-text'>
+        <p>
+          The <code>unipept pept2prot</code> command can be used in combination with Megan, for example to perform a functional analysis of the sample. This requires using the <code>--meganize</code> option that was added in version 1.2.0.
+        </p>
+        <h5>Example</h5>
+        <pre>
+<b>$</b> unipept pept2prot <b>--meganize</b> MDGTEYIIVK ISVAQGASK
+MDGTEYIIVK	ref|WP_008705701.1|	100	10	0	0	0	10	0	10	1e-100	100
+ISVAQGASK	ref|NP_003881.2 XP_011547112.1 XP_011547113.1|	100	10	0	0	0	10	0	10	1e-100	100</pre>
+        <p>
+          The generated output can be saved to a file and imported into Megan using the following settings:
+          <ul>
+            <li>Import from blast</li>
+            <li>Select the file with the <code>unipept pept2prot --meganize</code> output</li>
+            <li>Set format to blastTab and mode to blastp</li>
+            <li>Remove the fasta mapping that was automatically added</li>
+            <li>Enable the tabs you want (taxonomy/interpro2go/...), but always select the "use accession" option for this</li>
+            <li>Go back to the first tab and make sure is says blasttab and blastp, because sometimes it changes back when you switch tabs</li>
```

Comment:
make sure `s/is/it/` says


_[Original comment](https://github.ugent.be/unipept/unipept/pull/617#discussion_r12056) by @ninewise on Thu Aug 17 2017 at 15:54._